docusaurus/docs/iOS/swiftui/components/message-list.md

```diff
@@ -62,7 +62,16 @@ let messageListConfig = MessageListConfig(messageDisplayOptions: messageDisplayO
 let utils = Utils(messageListConfig: messageListConfig)
         
 let streamChat = StreamChat(chatClient: chatClient, utils: utils)
-``` 
+```
+
+## Date Indicators
+
+The SDK supports two types of date indicators - floating overlay and date separators inside the messages. This feature can be configured via the `dateIndicatorPlacement` in the `MessageListConfig`. With the floating overlay option (`.overlay`), the date indicator is shown for a short time whenever a new message appears. On the other hand, if you want to always show the date between messages, similarly to Apple Messages and WhatsApp, you should use the `.messageList` option. You can turn off both options by using the `.none` option. Here's an example how to setup the `messageList` option:
+
```

Comment:
Everything looks great, the only thing I would add is that I would highlight the existence of `dateLabelSize` on the `MessageListConfig` explanation so that users know the location where they define the height of the top padding.

Reply:
That's a great point, thanks for bringing it up 👍